src/model/json-schema.json

```diff
@@ -3628,10 +3628,7 @@
                     }
                 },
                 "abTests": {
-                    "type": "object",
-                    "additionalProperties": {
-                        "type": "string"
-                    }
+                    "$ref": "#/definitions/Record<string,\"control\"|\"variant\">"
```

Comment:
I think this is ok? Looks like recently the type for this property was update https://github.com/guardian/dotcom-rendering/pull/3276/files

Reply:
i'm not sure I understand these updates

Reply:
changes to this file are now ignored :)